src/app/settings/components/settings/settings-menu/settings-menu.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-toolbar class="c-side-bar">
```

Comment:
We have been talking about (but not done much about it) that we would like to "abstract away" the dependency of component libraries, so that if we decide to change them, we wouldn't have to change our entire codebase.

We are currently not 100% set in stone that we want to use Angular Material.
So it would be nice if we could make our own `ToolbarComponent` etc. These components would for now just use Angular Material. So the `SofieToolbarComponent` would just use `<mat-toolbar>...</mat-toolbar>` in its markup.

The benenfit would be that our entire frontend codebase wouldn't be dependent on a specific component library, but just our own components. So if we decide to change component library, we would only need to update those components and not our entire frontend.

I'm curious to hear your input on this suggestion :)